BinaryObjectScanner/Protection/SecuROM.cs

```diff
@@ -341,5 +342,138 @@ private static string GetV8WhiteLabelVersion(PortableExecutable exe)
 
             return null;
         }
+        
+        /// <summary>
+        /// Helper method to run checks on a SecuROM Matroschka Package
+        /// </summary>
+        private static string? CheckMatroschkaPackage(PortableExecutable exe, bool includeDebug)
+        {
+            var matroschka = exe.MatroschkaPackage;
```

Comment:
Just for consistency, I would prefer `package` as the name of the object.

Reply:
Fixed